dex/elrond_dex_pair/src/config.rs

```diff
@@ -26,10 +26,6 @@ pub trait ConfigModule {
     #[storage_mapper("extern_swap_gas_limit")]
     fn extern_swap_gas_limit(&self) -> SingleValueMapper<Self::Storage, u64>;
 
-    #[view(getTranferExecGasLimit)]
```

Comment:
why this view function is removed?

Reply:
Some simplification in the framework functions. ExecuteOnDest context no longer needs gas limit and will use the gas available for the whole transaction